console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -1977,7 +1977,9 @@ function deleteIotProject(iotProject) {
   let devIndex = getIotDevicesProjectIndex(iotProject.name);
 
   // delete iot devices for this project
-  iotdevices[devIndex].splice(devIndex, 1);
+  if (devIndex >= 0 && iotdevices[devIndex].length > 0) {
+    iotdevices[devIndex].splice(devIndex, 1);
+  }
```

Comment:
maybe raise an error like device not found when devIndex is null ?

Reply:
It was throwing error on delete project. I can revert this check if below error fix.

![image](https://user-images.githubusercontent.com/11775081/86349770-4a33d200-bc7f-11ea-9140-bb0053e321bb.png)